src/elements/hx-password-control/index.js

```diff
@@ -0,0 +1,28 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
+
+/**
+ * Defines behavior for the `<hx-pawword-control>` element.
```

Comment:
Please correct to `<hx-password-control>`.

Reply:
Changes have been done, please review.